### PR TITLE
Added Portuguese manpage

### PR DESCRIPTION
**Pull request by ProgEsteves**
Translation manpage to Portuguese
Refs https://github.com/suve/LD25/issues/1

### CODE REVIEW

File: pkg/colorful-portuguese.man [hunk 1]
```diff
@@ -0,0 +1,31 @@
+.\" Manpage para colorful
+.\" Entre em contato com veg@svgames.pl para corrigir erros de digitação.
+.TH man 6 "2014-03-16" "1.2" "Games Manual"
```
Comment (suve) [inline pkg/colorful-portuguese.man]: "Games Manual" should also be translated. The date should also reflect the file modification date.

**Review by suve — CHANGES_REQUESTED** [review]
Apart from those two minor issues, everything seems good.

**Comment by ProgEsteves** [comment]
Fixed

File: pkg/colorful-portuguese.man (hunk 1 above)
Comment (ProgEsteves) [reply pkg/colorful-portuguese.man]: Ok... It has been fixed.

**Review by suve — APPROVED** [review]

**Comment by suve** [comment]
All good now. Merged. Thanks!

**Pull request merged**